ledger/ledger-runner-common/src/main/scala/com/daml/ledger/runner/common/CliConfig.scala

```diff
@@ -84,7 +84,7 @@ object CliConfig {
       commandConfig = CommandConfiguration.Default,
       eventsPageSize = AcsStreamsConfig.DefaultEventsPageSize,
       bufferedStreamsPageSize = IndexServiceConfig.DefaultBufferedStreamsPageSize,
-      eventsProcessingParallelism = IndexServiceConfig.DefaultEventsProcessingParallelism,
+      bufferedEventsProcessingParallelism = IndexServiceConfig.DefaultEventsProcessingParallelism,
```

Comment:
let's stick with the pattern. It's easier in search later
```suggestion
      bufferedEventsProcessingParallelism = IndexServiceConfig.DefaultBufferedEventsProcessingParallelism,
```

Reply:
Ok, done